docs/index.rst

```diff
@@ -28,6 +28,7 @@ torchgeo
    tutorials/getting_started
    tutorials/trainers
    tutorials/benchmarking
+   tutorials/indices
```

Comment:
I haven't looked through the tutorial notebook yet but I would like it to be a generic "here is how to use both Kornia and torchgeo transforms separately and together" instead of just a tutorial on indices. So the file should be named `transforms.ipynb`.

Reply:
The tutorial shows how to use the functional indices (e.g. (ndvi), the indices modules (e.g. AppendNDVI), and chaining the indices modules together using the AugmentationSequentialWrapper and other kornia augmentations and applying them to a batch dict of imagery.

The reason I sertled on the indices title is because I also added an additional section on how to extract and download a smaller region of interest from Sentinel-2 COGs and then us our functional indices to compute indices and then visualize them.

It might make more sense to split them into two separate tutorials though.

Reply:
I split them into separate notebooks, indices.ipynb and transforms.ipynb.

Reply:
I don't think we need a notebook for each type of transform, what do you think @calebrob6?

Reply:
I was thinking of indices as their own thing since, if you look at the notebook, it's centered around using them to create index visualizations on raw Sentinel-2 imagery. The transforms notebook I was thinking would just be for all other general transforms.